ert3/algorithms/_sensitivity.py

```diff
@@ -11,19 +11,22 @@ def _build_base_records(
 
 
 def one_at_the_time(
-    parameters: MutableMapping[str, Distribution]
+    parameters: MutableMapping[str, Distribution], tail: Optional[float] = None
```

Comment:
Why not do the following instead?

```suggestion
    parameters: MutableMapping[str, Distribution], tail: float = 0.99
```